splunk_add_on_ucc_framework/app_conf.py

```diff
@@ -36,6 +36,7 @@ def update(
         conf_file_names: Sequence[str],
         is_visible: bool,
         check_for_updates: bool = True,
+        **kwargs: str,
```

Comment:
If you want to do `kwargs`, let's put `check_for_updates` there as well to be consistent. It's an internal code, we can change it if we need to.

Reply:
done